dts/arm/st/f2/stm32f2.dtsi

```diff
@@ -172,7 +172,7 @@
 		usart1: serial@40011000 {
 			compatible = "st,stm32-usart", "st,stm32-uart";
 			reg = <0x40011000 0x400>;
-			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00004000>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00000010>;
```

Comment:
Already fixed in https://github.com/zephyrproject-rtos/zephyr/pull/32028 based on the issue you reported.
Commit could be removed.

Reply:
I'm sorry, I'm kind of new to all these things about pull requests.
I'll try my best.
I'm going to remove the clock commit.